src/typings.d.ts

```diff
@@ -31,3 +31,7 @@ interface AutocompletionOptions {
   path: string;
   size: number;
 }
+
+interface AppConfig {
```

Comment:
Was thinking that at the moment the config is for jsonschema properties, but we might have some other config to drive the behaviour of the editor (and not tight to a field path) so we will expand this interface.